### ISSUE TITLE
Make FormData a cloneable object

### ISSUE THREAD

**Issue by annevk** [whatwg/xhr#55]
This can be done after https://github.com/whatwg/html/pull/727 lands.

**Comment by wisniewskit** [comment]
Was this something that should be dealt with now that the referenced landing happened?

**Comment by annevk** [comment]
Yeah maybe, although I haven't seen much interest. Do you happen to know if in the implementation we effectively clone `FormData` objects when they are passed to `XMLHttpRequest`?

**Comment by inexorabletash** [comment]
FWIW, in Blink's XHR implementation we synchronously serialize the passed FormData; nothing shared with the cloning implementation, no script-visible side effects, etc.

**Comment by annevk** [comment]
Apparently Firefox already supports this. Is there interest from Chrome in supporting this @inexorabletash? @youennf WebKit? @travisleithead Edge?

**Comment by inexorabletash** [comment]
SGTM; not high on the priority list but seems doable. Tests would be great.

**Comment by jed-meyer** [comment]
> Apparently Firefox already supports this.

This doesn't hold true (anymore), see dfahlander/Dexie.js#861

**Comment by annevk** [comment]
Well,
```js
onmessage = e => console.log(e.data)
postMessage(new FormData(),"*");
```
does log a `FormData` object in Firefox Nightly and an exception in Chrome Canary. However, the serialization steps for a given platform object are also passed a _forStorage_ boolean. I suspect Firefox throws if that is true, based on your analysis.

**Comment by pavinduLakshan** [comment]
Is this now supported?
